# /database/srv/GetPotList.srv
# 花盆信息列表

---
PotInfo[] pots
